<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>121</y>
      <width>381</width>
      <height>351</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="Number7Button">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>51</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number8Button">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>51</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number4Button">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>126</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number5Button">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>125</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number2Button">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>200</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number0Button">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>275</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DotButton">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>274</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number1Button">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>200</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResultButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>274</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number6Button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>125</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number9Button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>51</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Number3Button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AdditionButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>294</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SubtractButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>237</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ProductButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>181</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DivisionButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>125</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>รท</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackspaceButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>68</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>โซ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AcButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: #d9d9d9

}
</string>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>3</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

					</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="ScreenFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>353</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QLineEdit" name="ScreenlineEdit">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>6</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Digital-7</family>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>401</width>
      <height>471</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_4</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
